src/_data/shells.yml

```diff
@@ -1,14 +1,14 @@
 - name: bash
```

Comment:
Might it be that Flutter is NOT in /usr/bin?

Reply:
True, but this is consistent with the previous advice given on the page, which is to consider adding `flutter` to `/usr/bin`. If they follow inline, this should be OK.